.travis.yml

```diff
@@ -14,9 +14,13 @@ env:
     - ROS_DISTRO="kinetic" NOT_TEST_BUILD=true ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu VERBOSE_OUTPUT=false
     - ROS_DISTRO="kinetic" NOT_TEST_BUILD=true ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu DEBUG_BASH=true
     - ROS_DISTRO="kinetic" PRERELEASE=true
+    - ROS_DISTRO="melodic" NOT_TEST_BUILD=true ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu VERBOSE_OUTPUT=false
+    - ROS_DISTRO="melodic" NOT_TEST_BUILD=true ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu DEBUG_BASH=true
+    - ROS_DISTRO="melodic" PRERELEASE=true
```

Comment:
I also added melodic to the CI now.  I haven't technically tested this on `kinetic`, so CI will help determine success there, too.